Library/Homebrew/gpg.rb

```diff
@@ -38,12 +37,21 @@ def self.create_test_key(path)
       Key-Length: 2048
       Subkey-Type: RSA
       Subkey-Length: 2048
-      Passphrase: ''
       Name-Real: Testing
       Name-Email: testing@foo.bar
       Expire-Date: 1d
+      %no-protection
       %commit
     EOS
     system GPG_EXECUTABLE, "--batch", "--gen-key", "batch.gpg"
   end
+
+  def self.cleanup_test_processes
```

Comment:
`self.cleanup_test_processes!` probably because it's going to kill things.

Reply:
Sure, will change.